Core/OfficeDevPnP.Core/Framework/Provisioning/ObjectHandlers/ObjectListInstance.cs

```diff
@@ -1467,7 +1467,8 @@ private static XElement PrepareField(XElement fieldElement)
                     isDirty = false;
                 }
 
-                if (existingList.ContentTypesEnabled)
+                //it might make sense to update ContentType on existing list in case we want to add/refresh a ContentType even if users should not choose
+                if (existingList.ContentTypesEnabled|| UpdateContentTypeOnExistingLists)
```

Comment:
Maybe the new property shouldn't be called `UpdateContentTypeOnExistingLists` but something like `ForceUpdateListContentType` or so. `UpdateContentTypeOnExistingLists` sounds like I have to to always set it to `true` because otherwise CTs won't be updated. But this is not the case. They _are_ updated, but only if CTs are enabled on the list.
Or what about `UpdateContentTypesOnListsWithDisabledContentTypes`?

Naming is hard :-/

Reply:
Hi- it could be "ForceUpdateListCTWhenManageCTDisabled" - but maybe a better comment with the parameter would be sufficient as well.